htpasswd/auth.go

```diff
@@ -94,17 +98,24 @@ func (a *Auth) Authenticate(username, password string) (bool, error) {
 
 // Reload htpasswd file if it changed during current run
 func reloadIfChanged(a *Auth) {
-	fileInfo, err := os.Stat(a.filename)
-	if err != nil {
-		//On error, retain current file
-		return
+	reload := false
+	currentmodTime := a.modTime
```

Comment:
If you call `parse(); return;` in the second if, you don't need this variables at all.